README.md

```diff
@@ -170,6 +170,13 @@ $ dotenv-linter --exclude .env.test
 .env:3 UnorderedKey: The BAR key should go before the FOO key
 ```
 
+If you need to skip some checks, you can use the argument `--skip CHECK_NAME` or its short version `-s CHECK_NAME` (will be available in [v2.0.0](https://github.com/dotenv-linter/dotenv-linter/issues/172)):
+
+```shell script
+$ dotenv-linter --skip UnorderedKey EndingBlankLine
```

Comment:
In order to have an easier usage for the user of the tool it would be great to have a list of that possible checks somewhere without the need to look at the code

Reply:
Agree. I have created the issue https://github.com/dotenv-linter/dotenv-linter/issues/199 and have included it to the next release.